pkg/metrics/reconciliation_status.go

```diff
@@ -0,0 +1,56 @@
+package metrics
+
+import (
+	"fmt"
+
+	"github.com/kyma-incubator/reconciler/pkg/cluster"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	prometheusSubsystem = "reconciler"
+)
+
+// ReconciliationStatusCollector provides the following metrics:
+// - reconciler_reconciliation_status{"cluster_id", "runtime_id", "global_account_id", "configuration_id"}
+// These gauges show the status of the reconciliation.
+// The value of the gauge could be:
+// 0 - Error
```

Comment:
Not sure if we still need this comment after we moved it to the cluster-status object.

Reply:
You're right, we don't need this here.